src/components/Page/components/Header/Header.scss

```diff
@@ -52,57 +50,41 @@ $action-menu-rollup-computed-width: rem(40px);
 }
 
 .BreadcrumbWrapper {
-  flex: 0 1 auto;
+  flex: 0 0 auto;
+  grid-area: breadcrumbs;
   max-width: 100%;
-
-  &.newDesignLanguage {
-    grid-area: breadcrumbs;
-    flex: 0 0 auto;
-    margin-right: spacing();
-  }
+  margin-right: spacing();
 
   @include print-hidden;
 }
 
 .PaginationWrapper {
-  flex: 0 0 auto;
```

Comment:
Should this stay?

Reply:
Added this back. I think it was removed by accident so good catch 👍